source/fsicalmanagement/dataaccess/user_repository.d

```diff
@@ -0,0 +1,252 @@
+module fsicalmanagement.dataaccess.user_repository;
+
+import fsicalmanagement.model.user : User;
+import poodinis : Value;
+import std.algorithm : map;
+import std.conv : to;
+import std.typecons : Nullable, nullable;
+import std.range.interfaces : InputRange, inputRangeObject;
+
+/**
+ * A repository which stores `User`s.
+ */
+interface UserRepository
```

Comment:
In this file, the same concerns as in `event_repository.d` apply.

Reply:
done